source/Nuget.nanoFramework.Hardware.ESP32/Nuget.nanoFramework.Hardware.ESP32.nuproj

```diff
@@ -0,0 +1,74 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="12.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <ItemGroup Label="ProjectConfigurations">
+    <ProjectConfiguration Include="Debug|AnyCPU">
+      <Configuration>Debug</Configuration>
+      <Platform>AnyCPU</Platform>
+    </ProjectConfiguration>
+    <ProjectConfiguration Include="Release|AnyCPU">
+      <Configuration>Release</Configuration>
+      <Platform>AnyCPU</Platform>
+    </ProjectConfiguration>
+  </ItemGroup>
+  <ItemGroup>
+    <Content Include="..\bin\$(Configuration)\Windows.Devices.Gpio.dll">
+      <Link>lib\Windows.Devices.Gpio.dll</Link>
+    </Content>
+    <Content Include="..\bin\$(Configuration)\Windows.Devices.Gpio.pdb">
+      <Link>lib\Windows.Devices.Gpio.pdb</Link>
+    </Content>
+    <Content Include="..\bin\$(Configuration)\Windows.Devices.Gpio.pdbx">
+      <Link>lib\Windows.Devices.Gpio.pdbx</Link>
+    </Content>
+    <Content Include="..\bin\$(Configuration)\Windows.Devices.Gpio.pe">
+      <Link>lib\Windows.Devices.Gpio.pe</Link>
+    </Content>
+    <Content Include="..\bin\$(Configuration)\Windows.Devices.Gpio.xml">
+      <Link>lib\Windows.Devices.Gpio.xml</Link>
+    </Content>
+  </ItemGroup>
+  <ItemGroup>
+    <Folder Include="lib\" />
+  </ItemGroup>
+  <ItemGroup>
+    <Dependency Include="nanoFramework.CoreLibrary">
+      <Version>[1.0.0-preview042]</Version>
+    </Dependency>
+  </ItemGroup>
+  <ItemGroup>
+    <Dependency Include="nanoFramework.Runtime.Events">
+      <Version>[1.0.0-preview167]</Version>
+    </Dependency>
+  </ItemGroup>
+  <ItemGroup>
+    <Dependency Include="nanoFramework.System.DateTime">
+      <Version>[1.0.0-preview007]</Version>
+    </Dependency>
+  </ItemGroup>
+  <PropertyGroup Label="Globals">
+    <ProjectGuid>96eaba11-6a33-454d-a7b2-b96ca5617e8c</ProjectGuid>
```

Comment:
Please replace this GUID, it's been used in the equivalent Nuget project in mscorlib repo

Reply:
Also changed the DELIVERABLES one as well, just in case.
I thought there would be a issue with GUIDs, especially starting from copied projects.